### PR TITLE
Remove vendor and pkg

### PR DESCRIPTION
**Pull request by sjmiller609**
Part 1/2 for Multi stage docker builds in observance of https://github.com/astronomer/commander/pull/14#issuecomment-541156307

### CODE REVIEW

**Comment by sjmiller609** [comment]
@andriisoldatenko @schnie 
any action required here?

**Comment by andriisoldatenko** [comment]
@sjmiller609 can you pls fix build?

**Comment by sjmiller609** [comment]
the build is failing because there is no circle CI config file. that will be added with the component testing

File: Dockerfile [hunk 1]
```diff
@@ -12,37 +12,44 @@ LABEL io.astronomer.docker.component="houston"
 LABEL io.astronomer.docker.environment="development"
```
Comment (schnie) [inline Dockerfile]: @sjmiller609 maybe flip this to commander

Comment (sjmiller609) [reply Dockerfile]: that's in the next PR

**Pull request merged**